src/core/condense/index.ts

```diff
@@ -8,6 +8,8 @@ import { ApiMessage } from "../task-persistence/apiMessages"
 import { maybeRemoveImageBlocks } from "../../api/transform/image-cleaning"
 
 export const N_MESSAGES_TO_KEEP = 3
+export const MIN_CONDENSE_THRESHOLD = 10 // Minimum percentage of context window to trigger condensing
```

Comment:
We changed to 5% in main - I think it makes sense to match that.

```suggestion
export const MIN_CONDENSE_THRESHOLD = 5 // Minimum percentage of context window to trigger condensing
```